radon/tests/test_raw.py

```diff
@@ -246,7 +246,7 @@ def foo(n=1):
                 This is a string not a comment
                 """
         test = 0
-    ''', (6, 6, 9, 1, 3, 0, 0)),
+    ''', (9, 5, 6, 1, 3, 0, 0)),
```

Comment:
I've fixed up the tests.  This is actually because I failed to specify `r` for the string so the backslash character was never passed to the test.